tests/Fixtures/DocBlockType/VirtualPropertyGetter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JMS\Serializer\Tests\Fixtures\DocBlockType;
+
+use JMS\Serializer\Annotation as Serializer;
+
+class VirtualPropertyGetter
+{
+    /**
+     * @Serializer\VirtualProperty
+     *
+     * @return string[]
+     */
+    #[Serializer\VirtualProperty]
+    public function getArrayOfIntegers(): array
```

Comment:
Name seems to be a bit misleading. :) Can we rename it?

Reply:
Sorry I changed the type of value while doing it, forgot to change the method name.

Done !